src/diffusers/pipelines/stable_diffusion/convert_from_ckpt.py

```diff
@@ -17,32 +17,57 @@
 import os
 import re
 import tempfile
-from typing import Optional
+from typing import Optional, Type
+from pathlib import Path
 
 import requests
 import torch
 from transformers import AutoFeatureExtractor, BertTokenizerFast, CLIPTextModel, CLIPTokenizer, CLIPVisionConfig
 
-from diffusers import (
+import requests
+
+from transformers import (
+    AutoFeatureExtractor,
+    BertTokenizerFast,
+    CLIPTextModel,
+    CLIPTokenizer,
+    CLIPVisionConfig,
+    logging,
+)
+
+from huggingface_hub import snapshot_download
+
+from ...models import (
     AutoencoderKL,
+    UNet2DConditionModel,
+)
+from ...schedulers import (
     DDIMScheduler,
     DPMSolverMultistepScheduler,
     EulerAncestralDiscreteScheduler,
     EulerDiscreteScheduler,
     HeunDiscreteScheduler,
-    LDMTextToImagePipeline,
     LMSDiscreteScheduler,
     PNDMScheduler,
-    StableDiffusionPipeline,
-    UNet2DConditionModel,
 )
-from diffusers.pipelines.latent_diffusion.pipeline_latent_diffusion import LDMBertConfig, LDMBertModel
-from diffusers.pipelines.paint_by_example import PaintByExampleImageEncoder, PaintByExamplePipeline
-from diffusers.pipelines.stable_diffusion import StableDiffusionSafetyChecker
 
-from ...utils import is_omegaconf_available, is_safetensors_available, logging
+from ..latent_diffusion.pipeline_latent_diffusion import LDMBertConfig, LDMBertModel
+from ..paint_by_example import PaintByExampleImageEncoder
+from .safety_checker import StableDiffusionSafetyChecker
+
+from ..pipeline_utils import DiffusionPipeline
+
+from ...utils import (
+    DIFFUSERS_CACHE,
+    HF_HUB_OFFLINE,
+    is_safetensors_available,
+    is_omegaconf_available,
+)
+
 from ...utils.import_utils import BACKENDS_MAPPING
 
+# silences transformers warning on CLIP Model loading unused keys
+logging.set_verbosity_error()
```

Comment:
```suggestion
```

Silencing errors should be job of script that calls this script. Agreed the CLIP unused key errors are annoying :)